README.md

```diff
@@ -107,3 +107,27 @@ Things you may want to cover:
 >  }
 >}
 >```
+
+### Create User Wine Data
+`POST https://weathervine-be.herokuapp.com/api/v1/users/:user_id/wines`
```

Comment:
Double checking here.. do we want to have `:wine_id` at the end of the endpoint here?

Reply:
In this case, the provided ID in the URI would be the user_wine id, which technically _isn't_ created yet (until the post is successful). We would want the id at the end of this URI for a patch request on the user wine record.

Reply:
Ah.. thank you for the clarification here. Yes that makes sense because we haven't created the user_wine record yet.